packages/xstate-wallet/.vscode/launch.json

```diff
@@ -25,7 +25,7 @@
       "request": "launch",
       "args": [
         "node_modules/.bin/jest",
-        "--config=${workspaceRoot}/config/jest/jest.config.js",
+        "--config=${workspaceFolder}/config/jest/jest.config.js",
```

Comment:
What's this change related to?

Reply:
I open the entire repo as a folder in a workspace and then add each package individually as a workspace folder, so I have a multi-root workspace setup. So, when I press `F5` to run a test, I want the test to use the jest config file from the `workspaceFolder` that I currently have open, which in this case is `packages/xstate-wallet`.

Reply:
I _think_ it should work for people with the setup that is opening `packages/xstate-wallet` directly as root.